haris2055/SOEN390
# Refactored and redesigned navbar component

Refactored code for navbar and restyled it. 

To test: 
- click through the various links in the navbar to ensure that all sections of the navbar are correctly linked to their corresponding pages
- make sure that the correct navbar headers appear when the user is logged in vs when the user is logged out
